macro/src/main/scala/StructTypeEncoder.scala

```diff
@@ -39,10 +39,7 @@ case object FieldNull extends OptionalField[Nothing] {
 }
 
 object OptionalField {
-  implicit def fieldToOption[T: Manifest]: Transformer[OptionalField[T], Option[T]] =
-    new Transformer[OptionalField[T], Option[T]] {
-      override def transform(src: OptionalField[T]): Option[T] = src.toOption
-    }
+  implicit def fieldToOption[T: Manifest]: Transformer[OptionalField[T], Option[T]] = _.toOption
```

Comment:
also what difference does this make 😓

Reply:
It's just shorter. Sorry for doing mess in the PR, I just wanted to celebrate that we have Scala 2.12 now :sweat_smile: